src/backend/SyncBackendConnection.cpp

```diff
@@ -627,4 +627,18 @@ std::vector<StatisticsData> SyncBackendConnection::get_data(
     }
 }
 
+void SyncBackendConnection::set_alias(
+        const backend::EntityId& id,
+        const std::string& new_alias)
+{
+    try
+    {
+        StatisticsBackend::set_alias(id, new_alias);
+    }
+    catch (const Exception& e)
+    {
+        qWarning() << "Fail setting new alias for entity: " << id.value();
```

Comment:
This could be in the issues list.

Reply:
we can create a call in Engine to handle all the backend possible issues and add them to issues (similar than callbacks).
It is a bit tricky though because the sync_backend does not have (and should not) a reference to Engine.
Let's add it in another PR.